src/screens/TaskDetailScreen.tsx

```diff
@@ -1,88 +1,53 @@
 // src/screens/TaskDetailScreen.tsx
-import React, { useState, useEffect, useCallback, useMemo } from 'react';
+import React, { useState } from 'react';
```

Comment:
You’re only importing useState but could pull in useCallback and useEffect up front. Wrapping handlers in useCallback and syncing props in useEffect will help avoid stale closures and satisfy lint rules.